### PR TITLE
hwmv2: Change board names to vendor prefixes

### PR DESCRIPTION
**Pull request by nordicjm**
I think in nearly all cases this makes things much worse but there you go

### CODE REVIEW

**Review by gmarull — APPROVED** [review]

**Comment by nordicjm** [comment]
CC @mah-eiSmart @jadonk

**Comment by mah-eiSmart** [comment]
CC @wm-eisos Please keep notice of that

**Review by rerickson1 — APPROVED** [review]

**Comment by nordicjm** [comment]
@golowanow this has a commit which fixes up intel_adsp

**Comment by golowanow** [comment]
> @golowanow this has a commit which fixes up intel_adsp

@nordicjm, thank you

File: soc/intel/intel_adsp/ace/Kconfig [hunk 1]
```diff
@@ -13,16 +12,20 @@ config SOC_SERIES_INTEL_ADSP_ACE
...
+
+config SOC_INTEL_ACE15_MTPM
+	bool
+	help
+	  ACE 1.5 Meteor Lake PCH M
+
+config SOC_INTEL_ACE20_LNL
+	bool
+	help
+	  ACE 2.0 Lunar Lake PCH
```
Comment (tejlmand) [inline soc/intel/intel_adsp/ace/Kconfig:31]: why ?
Those are defined with `bool`s here:
https://github.com/zephyrproject-rtos/zephyr/blob/d87086055b5fa44d951e6e9a8b9f87c691a39af5/soc/intel/intel_adsp/ace/Kconfig.soc#L11-L17

**Review by tejlmand — CHANGES_REQUESTED** [review]

File: soc/intel/intel_adsp/ace/Kconfig (hunk 1 above)
Comment (nordicjm) [reply soc/intel/intel_adsp/ace/Kconfig:31]: Will address post-rebase to unblock vendors

**Review by tejlmand — APPROVED** [review]
Agreed to do https://github.com/zephyrproject-rtos/zephyr/pull/69285#discussion_r1500558049 as followup.

Regarding this rename, for some vendors this looks nice, but for some, like `brcm` vs. `broadcom` then I feel `broadcom` is cleaner, however this PR is what was agreed at arch wg.

Should we have this PR brought up at Arch wg, so more people are aware what the impact of the decision is ?

**Comment by carlescufi** [comment]
Architecture WG:

- Aligning folder names for vendors with vendor-prefixes.txt is accepted, no objections

**Pull request merged**